server.js

```diff
@@ -1,9 +1,30 @@
 let express = require('express');
 let app = express();
+var proxy = require('express-http-proxy');
+require('dotenv').config();
+// const { createProxyMiddleware } = require('http-proxy-middleware');
 
+app.use('/proxy', proxy('https://app-hrsei-api.herokuapp.com/', {
+  proxyReqOptDecorator: function(proxyReqOpts, srcReq) {
+    // you can update headers
+    proxyReqOpts.headers['Authorization'] = `${process.env.TOKEN}`;
```

Comment:
could we make a note in the readme of what config variables this needs

Reply:
All three should be fixed. Appreciate the feedback!